query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala

```diff
@@ -410,7 +410,10 @@ class QueryResource(secondary: Secondary,
   private def parseFuseColumnMap(s: String): Map[String, String] = {
     s.split(';')
      .map { item => item.split(',') }
-     .map { case Array(a, b) => (a, b) }
+     .map { case Array(a, b) => (a, b)
+            case _ =>
+               throw new Exception(s"X-Socrata-Fuse-Columns parse error $s.")
+     }
```

Comment:
Will this stop the com.socrata.http.client.exceptions.UnexpectedContentType: Wanted content-type application/json; got text/html error we see in SF?

Reply:
no, this isn't trying to fix the problem.  just want to get more information out.  I am hoping that the problem is obvious once we see the value in X-Socrata-Fuse-Columns